src/app.coffee

```diff
@@ -32,6 +31,8 @@ knex.init.then ->
 
 		bootstrap.done
 		.then ->
+			# The device module communicates with the api and as such needs bootstrapping to complete
+			device = require './device'
```

Comment:
The parts that communicate with the API already happen after `bootstrap.done`, i.e. [getID](https://github.com/resin-io/resin-supervisor/blob/master/src/device.coffee#L21), and then [applyState](https://github.com/resin-io/resin-supervisor/blob/master/src/device.coffee#L182) which depends on getID.
So it should be fine to leave the require where it was, unless I'm missing something else?